src/module.cc

```diff
@@ -78,7 +78,7 @@ NAN_METHOD(GetReport) {
   std::ostringstream out;
 
   GetNodeReport(isolate, kJavaScript, "JavaScript API", __func__, out);
-  // Return value is the NodeReport filename
+  // Return value is the contents of a NodeReport as a string.
```

Comment:
If #52 is merged first I'll need to make changes here, if this one is merged first then the renaming one will need updates. I'll make sure it gets done at that point with everything else rather than do just one of the (many) changes I'll probably need to make ahead of time.